downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -57,6 +57,7 @@ class Git : GitBase() {
             // Configure JGit to connect to the SSH-Agent if it is available.
             val sessionFactory = object : JschConfigSessionFactory() {
                 override fun configure(hc: OpenSshConfig.Host, session: Session) {
+                    session.setConfig("StrictHostKeyChecking", "no")
```

Comment:
Maybe say in the commit message that this is only a work-around until we find out why the `known_hosts` file is not respected?

Reply:
BTW, looks like you need to specific the `known_host` file explicitly like in the first example over [here](https://smarterco.de/JSch-without-known-hosts-check/).

Reply:
Hmm, in fact JGit's `JschConfigSessionFactory` should do that automatically it seems, but https://stackoverflow.com/a/15849893/1127485 has some hints.

Reply:
Done.

Reply:
Im not sure, this does not sound like you need to configure the location of the known hosts file:

> Normally you would think that it is enough to comment out the jsch.setKnownHosts(sshKnownHosts); line, but this would be too easy, right? ;) It still will check the known_hosts-file and throw some errors, if something goes wrong.